CHANGELOG.md

```diff
@@ -4,6 +4,266 @@ This changelog goes through all the changes that have been made in each release
+  * Tags cleanup (#13428)
+  * Various style tweaks (#13418)
+  * Refactor push update (#13381)
+  * Comment box tweaks and SVG dropdown triangles (#13376)
+  * Various style fixes (#13372)
+  * Change repo home page icons to SVG (#13364)
+  * Use CSS Vars for primary color (#13361)
+  * Refactor image paste code (#13354)
+  * Switch from SimpleMDE to EasyMDE (#13333)
+  * Group Label Changed Comments in timeline (#13304)
+  * Make the logger an interface (#13294)
+  * Fix PR/Issue titles on mobile (#13292)
+  * Rearrange the order of the merged by etc. in locale (#13284)
+  * Replace footer and modal icons with SVG (#13245)
+  * Issues overview should not show issues from archived repos (#13220)
+  * Show stale label for stale code comment which is marked as resolved (#13213)
+  * Use CSS Variables for fonts, remove postcss-loader (#13204)
+  * Add mentionable teams to tributeValues and change team mention rules to gh's style (#13198)
+  * Move install pages out of main macaron routes (#13195)
+  * Update outdated label to use Fomantic UI style (#13181)
+  * Added option to disable webhooks (#13176)
+  * Change order of possible-owner organizations to alphabetical (#13160)
+  * Log IP on SSH authentication failure for Built-in SSH server (#13150)
+  * Added option to disable migrations (#13114)
+  * New "Add Mirror" Button in the Organization view (#13105)
+  * Manually approve new registration (#13083)
+  * Cron job to cleanup hook_task table (#13080)
+  * Use the headline comment of pull-request as the squash commit's message (#13071)
+  * Clarify the suffices and prefixes of setting.AppSubURL and setting.AppURL (#12999)
+  * Slightly simplify the queue settings code to help reduce the risk of problems (#12976)
+  * Add precise search type for Elastic Search (#12869)
+  * Move APIFormat functions into convert package (#12856)
+  * Multiple GitGraph improvements: Exclude PR heads, Add branch/PR links, Show only certain branches, (#12766)
+  * Add TrN for repository limit (#12492)
+  * Refactor doctor (#12264)
+  * Add the tag list page to the release page (#12096)
+  * Redirect on changed user and org name (#11649)
+  * load U2F js only on pages which need it (#11585)
+  * Make archival asynchronous (#11296)
+  * Introduce go chi web framework as frontend of macaron, so that we can move routes from macaron to chi step by step (#7420)
+  * Improve vfsgen to not unzip bindata files but send to browser directly (#7109)
+  * Enhance release list (#6025)
+* DOCS
+  * Swagger show models by default (#14880)
+  * Add missing repo.projects unit into swagger (#14876)
+  * Update docs and comments to remove macaron (#14491)
+  * Issue template addition: Are you using Gitea behind CloudFlare? (#14098)
+  * Generate man pages (#13901)
+  * Reformat/fine-tune docs (#13897)
+  * Added Table of Contents to long documentation pages (#13890)
+  * Add docs command (#13429)
+  * Update external-renderers.en-us.md (#13165)
+* MISC
```

Comment:
```suggestion
* MISC
  * Move setting to enable closing issue via commit in non default branch to repo settings (#14965)
```
Note: 1.13 backport

Reply:
14965 was not backported, and since I dont know if it's realy a bug or an enhancement I wont do ...

never the less it is not MISC but either bug or enhancement

Reply:
It's a change to the permission model of repo settings. The way it behaved before was not really suitable for most usecases I suppose, so i'd say enhancement

Reply:
added